scripts/performance_testing/README.md

```diff
@@ -0,0 +1,211 @@
+# Performance Testing 
+
+Performance testing may be accomplished by using the performance testing tool found in this directory, `Test-ReleasePerformance.ps1`. Note that this script depends on other files from this repository. It may be run on external builds of Coding Standards through the `-CodingStandardsPath` flag, but it should be run from a fresh checkout of this repository. 
```

Comment:
would add the predicates capability with mention of the py script up here too, in case readers dont end up scrolling to the bottom, like an overview of the contents in this maybe

Reply:
Good idea!